__tests__/transaction/post.spec.ts

```diff
@@ -31,26 +45,39 @@ describe('Post Arweave Transaction', () => {
     expect(postedTxn.statusText).toBe('OK');
   });
 
-  it('should post a data transaction using bundlr', async () => {
-    // TODO
-    const key = JSON.parse(readFileSync('wallet1.json').toString());
-    const data = readFileSync('__tests__/transactions/data/test.json', 'utf-8');
-    const txn = await createTransaction({
-      data: data,
-      key: key,
-      options: { useBundlr: true },
+  it('should post a bundlr data transaction', async () => {
+    const generateWallet = await createWallet({
+      environment: 'local',
     });
 
+    writeFileSync('wallet1.json', JSON.stringify(generateWallet));
+
+    const { key } = JSON.parse(readFileSync('wallet1.json').toString());
+
+    const data = readFileSync('__tests__/transaction/data/test.json', 'utf-8');
+
+    const txn: BundlrTransaction = (await createTransaction({
+      key,
+      data,
+      type: 'data',
+      environment: 'local',
+      options: {
+        useBundlr: true,
+      },
+    })) as BundlrTransaction;
```

Comment:
Is defining both the variable type as a `BundlrTransaction` (line 59) as well as imposing the result of the function as the same (line 67) needed? Or can either one of them be removed. Will have to explain the same in docs accordingly.

Reply:
not really, did that for the tests. Was fighting a bug, if the input params are correct this shouldn't be an issue.
Matter of fact, taking it out